### DISCUSSION TITLE
vCon WG related IDs

### DISCUSSION THREAD

**Discussion by dgpetrie** [ietf-tools/datatracker#6741] [Help / Questions] [answered]
The vCon WG was just formed.  I see that the WG page picked up pre-existing draft-rosenberg-vcon-cc-usecases-00, but it did not pickup pre-existing draft-petrie-vcon-02.  Can you explain why or what needs to be done to add draft-petrie-vcon-02 as a related ID?

**Comment by rjsparks** [comment, marked as answer]
The glob for related drafts looks like

![image](https://github.com/ietf-tools/datatracker/assets/10996692/07f07eed-3528-4a3f-b02a-4997f97ff4d3)

That won't match `draft-petrie-vcon`

The chairs can add the draft to the group's managed list of documents.

**Reply by dgpetrie** [reply]
Awesome!  Thanks @rjsparks !

**Reply by Sarrac3873** [reply]
> Awesome!  Thanks @rjsparks !

**Comment by Kingcb1** [comment]
> The glob for related drafts looks like
> 
> ![image](https://github.com/ietf-tools/datatracker/assets/10996692/07f07eed-3528-4a3f-b02a-4997f97ff4d3)
> 
> That won't match `draft-petrie-vcon`
> 
> The chairs can add the draft to the group's managed list of documents.

**Comment by Sarrac3873** [comment]
> The vCon WG was just formed.  I see that the WG page picked up pre-existing draft-rosenberg-vcon-cc-usecases-00, but it did not pickup pre-existing draft-petrie-vcon-02.  Can you explain why or what needs to be done to add draft-petrie-vcon-02 as a related ID?
